Source/Private/LevelStatsCollector.cpp

```diff
@@ -1,24 +1,65 @@
 ﻿#include "LevelStatsCollector.h"
 
+#include "LevelStatsCollectorState.h"
+
 #include <Components/SceneCaptureComponent2D.h>
+#include <Engine/Engine.h>
 #include <Engine/LevelBounds.h>
 #include <Engine/TextureRenderTarget2D.h>
 #include <ImageUtils.h>
 
+FCustomPerformanceChart::FCustomPerformanceChart( const FDateTime & in_start_time, const FString & in_chart_label, const FString & in_output_path ) :
```

Comment:
you could pass by `FStringView` instead of `const FString &`

Reply:
bear in mind i was just following the exact same process as `ChartCreation.cpp` in the engine. It's changed anyways

Reply:
I understand that it's a better practice to use FStringView (didn't know that)